### PR TITLE
Build on main push

### PR DESCRIPTION
**Pull request by wawrzek**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -10,7 +10,33 @@ env:
...
+        with:
+          name: enclave_measurement
+          path: target/release/enclave_measurement.txt
+
+      - name: Adding measurement to summary
+        run: |
+          echo '### Enclave Image Measurement' >> $GITHUB_STEP_SUMMARY
+          cat target/release/enclave_measurement.txt >> $GITHUB_STEP_SUMMARY
+
       - name: Add software to the releases page
```
Comment (shamsasari) [inline .github/workflows/release.yml:40]: Maybe just simply called `Make release`

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -10,7 +10,33 @@ env:
...
     runs-on: ubuntu-latest
+
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: ./build.sh
+
+      - name: Service and enclave artifacts
```
Comment (shamsasari) [inline .github/workflows/release.yml:21]: Can you remove this step from the `build` workflow. It's not really needed now.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -1,8 +1,6 @@
 name: Build
 
 on:
-  push:
-    branches: [ "main" ]
   pull_request:
     branches: [ "main" ]
```
Comment (shamsasari) [inline .github/workflows/build.yml:5]: We should rename build.yml to something like pr.yml and release.yml to main.yml.

**Review by shamsasari — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (wawrzek) [reply .github/workflows/release.yml:21]: Do you mean from the new 'pr' one? (You are commenting the 'release.yaml' file).

I don't mind removing it.

Comment (shamsasari) [reply .github/workflows/release.yml:21]: Yes, from the new pr.yml

Comment (wawrzek) [reply .github/workflows/release.yml:21]: OK. Check again if all steps are fine.

**Review by shamsasari — APPROVED** [review]
LGTM

**Review by shamsasari — CHANGES_REQUESTED** [review]
Sorry, just saw this error on main: https://github.com/projectglove/glove-monorepo/actions/runs/9895463279/job/27335415729#step:2:12.

Are we missing something?

**Comment by wawrzek** [comment]
> Sorry, just saw this error on main: https://github.com/projectglove/glove-monorepo/actions/runs/9895463279/job/27335415729#step:2:12.
> 
> Are we missing something?

I think it requires a github.ref (or something like that), which is not present in a PR. Let's try to merge it and see if that works.

**Review by shamsasari — APPROVED** [review]

**Pull request merged**